adaptivefiltering/widgets.py

```diff
@@ -55,14 +55,15 @@ def __init__(self, schema, on_change=None):
         # Construct the widgets
         self._form_element = self._construct(schema, root=True, label=None)
 
-    @property
     def widget(self):
         """Return the resulting widget for further use"""
         return ipywidgets.VBox(self._form_element.widgets)
 
     def show(self):
         """Show the resulting combined widget in the Jupyter notebook"""
         display(self.widget)
+        w = ipywidgets.VBox(self._form_element.widgets)
+        display(w)
```

Comment:
You are double-displaying now, aren't you?

Reply:
i'm not even sure where this function is ever used if i'm honest.
I also can't remember when i wrote this or for what purpose